update the zero sequence terms of the transformation matrix

ASHAD1 opened the pull request:
This branch updates the zero-sequence current (I_0) to be `I_0 = (Ia + Ib + Ic)/3` . This is based on the textbook,
> "Vector control and dynamics of AC drives", D.W. Novotny and T.A Lipo. page 57-62.

ASHAD1 committed:
update the zero sequence terms

npetersen2 reviewed (commented):
I believe the update is correct and works for various values of C: i.e. `2/3` and `sqrt(2/3)`

@elsevers do you agree?

elsevers committed:
Simplify zsq transformations

- Remove unnecessary multiplication of C/C in `transform_clarke`
- Remove unnecessary multiplication of C/C in `transform_dqz_inverse`

elsevers committed:
Fix parenthese error

elsevers commented:
@ASHAD1 I agree with your transformations. I cleaned up the code a bit to my tastes. Could you please checkout this branch and make sure it still compiles? Report your findings back here. If successful, I will approve this PR.

ASHAD1 commented:
@elsevers the simplified transformation looks good. I was able to compile the code without errors. I think we can merge this branch. Thanks.

elsevers reviewed (approved):
Looks good.
